recipes/typescript/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "typescript" %}
+{% set version = "2.6.0" %}
+{% set sha256 = "293bc1532d3a0df795fd832f4bf76c501e92af501a35e41826e7af579239daaf" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Microsoft/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+  skip: True  # [win64]
```

Comment:
What is the reason for skipping win64?

Reply:
@mwcraig: ci check didn't pass for window 64, so for now I decided to skip. Since I have already used noarch: generic, I think I should remove that statement, and instead use [skip appveyor] as a commit message.

Reply:
ok, could you please  that and then if it turns out Windows needs work it can be fixed up in the feedstock.

Reply:
Thanks. I have removed the skip statement for windows build, and instead used [skip appveyor] in the commit message.